### PR TITLE
[PHP-3] Implement tasks from Arrays lesson

### PR DESCRIPTION
**Pull request by VladSmorodsky**

### CODE REVIEW

File: src/arrays/Arrays.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+class Arrays implements ArraysInterface
+{
+    /**
+     * @param array $input
+     * @return array
+     */
+    public function repeatArrayValues(array $input): array
+    {
+        $repetitivedArray = [];
+        for ($i = 0; $i < count($input); $i++) {
```
Comment (yderevianko-cartera) [inline src/arrays/Arrays.php:16]: if you have time try to rewrite it without using 2 for loops

File: src/arrays/Arrays.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    /**
+     * @param array $input
+     * @return int
+     */
+    public function getUniqueValue(array $input): int
+    {
+        $countElementsArray = array_count_values($input);
+        $lowestUniqueValue = 0;
+
+        foreach ($countElementsArray as $key => $value) {
```
Comment (yderevianko-cartera) [inline src/arrays/Arrays.php:34]: if you have time try to rewrite it using build in php functions

File: src/arrays/Arrays.php [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
+    /**
+     * @param array $input
+     * @return array
+     */
+    public function groupByTag(array $input): array
+    {
+        $transformedArray = [];
+        for ($i = 0; $i < count($input); $i++) {
```
Comment (yderevianko-cartera) [inline src/arrays/Arrays.php:53]: if you have time try to rewrite it using build in php functions

**Review by yderevianko-cartera — APPROVED** [review]

File: src/arrays/Arrays.php (hunk 1 above)
Comment (VladSmorodsky) [reply src/arrays/Arrays.php:17]: Code is rewritten

File: src/arrays/Arrays.php (hunk 2 above)
Comment (VladSmorodsky) [reply src/arrays/Arrays.php]: Code is rewritten using php functions

File: src/arrays/Arrays.php (hunk 3 above)
Comment (VladSmorodsky) [reply src/arrays/Arrays.php]: Added build in php functions

File: src/arrays/Arrays.php [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        return count($countElementsArray) > 0 ? min(array_keys($countElementsArray)) : 0;
+    }
+
+    /**
+     * @param array $input
+     * @return array
+     */
+    public function groupByTag(array $input): array
+    {
+        $transformedArray[] = array_map(function($inputItem) use (&$transformedArray) {
```
Comment (yderevianko-cartera) [inline src/arrays/Arrays.php:42]: Nice try but array_map is similar to for loop and now it looks very complicated to understand ))
As you can see array functions are not always better way than for loops hovewer it is good to understand them.

**Review by yderevianko-cartera — APPROVED** [review]

**Pull request merged**